pkgs/development/python-modules/htmllistparse/default.nix

```
@@ -0,0 +1,28 @@
+{ lib
+, stdenv
+, buildPythonPackage
+, fetchPypi
+, beautifulsoup4
+, html5lib
+, requests
+, fusepy
+}:
+
+buildPythonPackage rec {
+  pname = "htmllistparse";
+  version = "0.6.1";
```

```suggestion
  version = "0.6.1";
  pyproject = true;
```
Could you please add this parameter?
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#buildpythonpackage-parameters-buildpythonpackage-parameters

---

setting pyproject = true breaks the build

---

as does pyproject = false.

---

Please add `setuptools` to `nativeBuildInputs`.